packages/js/src/notifications/notifications.ts

```diff
@@ -0,0 +1,236 @@
+import { BaseModule } from '../base-module';
+import { ActionTypeEnum, NotificationFilter, Result } from '../types';
+import { archive, completeAction, read, revertAction, unarchive, unread } from './helpers';
+import { Notification } from './notification';
+import type {
+  ArchivedArgs,
+  CompleteArgs,
+  FiltersCountResponse,
+  ListNotificationsArgs,
+  ListNotificationsResponse,
+  InstanceArgs,
+  ReadArgs,
+  UnarchivedArgs,
+  UnreadArgs,
+  RevertArgs,
+  FilterCountArgs,
+  CountArgs,
+  FilterCountResponse,
+  FiltersCountArgs,
+  CountResponse,
+  BaseArgs,
+} from './types';
+import { NovuError } from '../utils/errors';
+
+export class Notifications extends BaseModule {
+  async list({ limit = 10, ...restOptions }: ListNotificationsArgs = {}): Result<ListNotificationsResponse> {
```

Comment:
The function to get notifications is now called `list`, also I made similar changes to preferences.